.github/workflows/report.yml

```diff
@@ -82,15 +82,58 @@ jobs:
         with:
           version: ${{ inputs.bowtie-version }}
 
-      - name: Generate implementations.json file
+      - name: Generate implementations.json (Private API) file
         run: |
           bowtie info $(bowtie filter-implementations | sed 's/^/-i /') --format json > implementations.json
 
+      - name: Generate implementations API file for JSON Schema Org
+        run: |
+          bowtie filter-implementations | \
+            while read -r impl; do
+              COMPLIANCE_URLS=( \
+                bowtie filter-dialects -i $impl | \
+                jq -R -s 'split("\n")[:-1] | map(. as $uri | input | fromjson | .[] | select(.uri == $uri) | \
+                "https://bowtie.report/badges/\($impl)/compliance/" + .shortName)' data/dialects.json
+              )
+
+              bowtie info -i $impl --format json | \
+              jq --arg impl "$impl" --argjson compliance "$COMPLIANCE_URLS" '
+                {(.source): (
+                  {
+                    id: $impl,
+                    dialects: .dialects,
+                    badges_urls: (
+                      {
+                        supported_versions: "https://bowtie.report/badges/\($impl)/supported_versions.json",
+                        compliance: $compliance
+                      }
+                    )
+                  }
+                )}
+              '
+            done | \
+          jq -s 'add' > public-implementations
+
+      - name: Validate public-implementations against its corresponding JSON Schema
+        run: |
+          VALIDATION=$(bowtie validate --expect valid -i python-jsonschema bowtie/schemas/api/v1/json-schema-org/implementations.json public-implementations)
+          SUMMARY=$(echo "$VALIDATION" | bowtie summary --show failures --format json)
```

Comment:
This seems like a bug, `bowtie summary --show failures` should probably exit with non-zero status when there are failures, and then all of this woud go away and you can just use `--format markdown` probably?

Can you at least file that issue perhaps.